ui/pages/settings/networks-tab/custom-content-search/custom-content-search.js

```diff
@@ -57,11 +58,9 @@ export default function CustomContentSearch({
             position="end"
             onClick={() => handleSearch('')}
           >
-            <i
-              className="fa fa-times networks-tab__imageclose"
-              width="17"
-              heigth="17"
-              title="Close"
+            <Icon
+              className="networks-tab__imageclose"
+              name={ICON_NAMES.CLOSE}
             />
           </InputAdornment>
         )}
```

Comment:
Checked in storybook looks good. Had to modify component code to view

<img width="720" alt="Screenshot 2023-02-06 at 3 48 35 PM" src="https://user-images.githubusercontent.com/8112138/217112760-273eb404-7691-4b74-b433-527365a521a3.png">